### ISSUE TITLE
Make colour temperature changes gradual rather than instantaneous

### ISSUE THREAD

**Issue by azvero** [moroen/IKEA-Tradfri-plugin#63]
When I use a Tradfri remote to change the colour temperature of a white spectrum bulb, the transition is nice and gradual, taking a couple of seconds.

When I use Domoticz to do the same, the colour change is abrupt.

I wonder if it would be possible to make IKEA-Tradfri-plugin perform colour temperature changes over an (ideally, configurable) period of time rather than instantaneously?

For context, I change the colour temperature of my lights in line with daylight, similarly to applications like f.lux. This works, but the changeover from daylight to dusk to nighttime etc would be much nicer if it were gradual instead of all lights in the house suddenly changing colour.

**Comment by moroen** [comment]
Yes, this should be possible using the current API, I'll look into it! :)

**Comment by azvero** [comment]
That'd be amazing, thank you! :)

**Comment by moroen** [comment]
If you try the development branch, it now has a predefined transition time for changing dimmer levels and color-temps, as a "proof of concept". And quite right, it's much nicer! :)

Implementing a configurable period of time should be trivial, I'll get around to it as soon as possible!

**Comment by azvero** [comment]
Oh wow, that was quick. I am currently travelling but am keen to play with this when I return. Thank you!

**Comment by moroen** [comment]
The latest commit to development supports configurable transition times, defined as tenth of a second. Please test and give feedback!

**Comment by azvero** [comment]
Thanks!

**Comment by basrieter** [comment]
I just upgraded to this dev version and this small changes makes it sooo much nicer to switch WB and dim. Great work!

**Comment by azvero** [comment]
I finally had a chance to give the new feature a try. It's amazing — thank you so much for implementing it!
